Simplenote/src/main/java/com/automattic/simplenote/NoteListFragment.java

```diff
@@ -961,6 +1043,46 @@ public void bindView(View view, Context context, Cursor cursor) {
         }
     }
 
+    @Override
+    public void onBeforeUpdateObject(Bucket<Preferences> bucket, Preferences object) {
+    }
+
+    @Override
+    public void onDeleteObject(Bucket<Preferences> bucket, Preferences object) {
+        if (isAdded()) {
+            requireActivity().runOnUiThread(new Runnable() {
+                @Override
+                public void run() {
+                    getSearchItems();
+                }
+            });
+        }
```

Comment:
if we invert the conditional we avoid the nesting. not sure what your style preferences are but I know that for me seeing fewer levels of nesting can help surface bugs and behaviors easier

```java
public void onDeleteObject(Bucket<Preferences> bucket, Preferences object) {
	if (!isAdded()) {
		return;
	}

	requireActivity().runOnUiThread(new Runnable() {
		@Override
		public void run() {
			getSearchItems();
		}
	});
}
```

Reply:
Java looks pretty bad here with the nesting, but I read this as, "If the fragment is added, then get the search items."  I think the early exit requires about the same cognitive load in this case.